user/pages/04.Reference/05.object-storage/docs.en.md

```diff
@@ -15,25 +15,27 @@ It stores and retrieves arbitrary unstructured data objects via an HTTP-based AP
 
 You can create the OpenStack API to generate credentials to access the SysEleven Stack Object Storage. You can then use the S3 API with various S3 clients and/or SDKs.
 
-## Supported S3 operations
-
-Feature                          | Supported |
----------------------------------|-----------|
-bucket HEAD/GET/PUT/DELETE       | yes       |
-object HEAD/GET/PUT/POST/DELETE  | yes       |
-multipart upload                 | yes       |
-bucket listing                   | yes       |
-input compression                | yes       |
-request signature authentication | yes       |
-bucket/object ACL                | partially*|
-predefined ACL groups            | yes       |
-bucket/IAM user ACP              | no        |
-object versions                  | no        |
-object expiration                | no        |
-regions and storage classes      | no        |
-encryption                       | no        |
-access logging                   | no        |
-website hosting configuration    | no        |
+## Supported S3 operations by backend
+
+Feature                          | Support in ceph               | Support in quobyte            |
+---------------------------------|-------------------------------|-------------------------------|
+bucket HEAD/GET/PUT/DELETE       | yes                           | yes                           |
+object HEAD/GET/PUT/POST/DELETE  | yes                           | yes                           |
+multipart upload                 | yes                           | yes                           |
+bucket listing                   | yes                           | yes                           |
+input compression                | yes                           | yes                           |
+request signature authentication | yes                           | yes                           |
+predefined ACL groups            | yes                           | yes                           |
+bucket/object ACL                | partially*                    | partially*                    |
+bucket policies                  | no                            | yes                           |
+object versions                  | no                            | yes                           |
+object expiration                | no                            | yes                           |
+encryption                       | no                            | yes                           |
+CORS                             | no                            | yes                           |
+bucket/IAM user ACP              | no                            | no                            |
+regions and storage classes      | no                            | no                            |
+access logging                   | no                            | no                            |
+website hosting configuration    | no                            | no                            |
```

Comment:
CORS is part of this, we should group them and clarify what exactly, if not everything from this category, is supported.